disnake/client.py

```diff
@@ -741,22 +742,39 @@ async def login(self, token: str) -> None:
         data = await self.http.static_login(token.strip())
         self._connection.user = ClientUser(state=self._connection, data=data)
 
-    async def connect(self, *, reconnect: bool = True) -> None:
+    async def connect(
+        self, *, reconnect: bool = True, ignore_session_start_limit: bool = False
+    ) -> None:
         """|coro|
 
         Creates a websocket connection and lets the websocket listen
         to messages from Discord. This is a loop that runs the entire
         event system and miscellaneous aspects of the library. Control
         is not resumed until the WebSocket connection is terminated.
 
+        If the client has no more connects remaining in the current 24-hour window and ``ignore_session_start_limit`` is ``False``,
```

Comment:
```suggestion
        If the client doesn't have enough connects remaining in the current 24-hour window and ``ignore_session_start_limit`` is ``False``,
```
Generalizing the text a bit more since it's also used in `AutoShardedClient`